common/src/api/internal/nexus.rs

```diff
@@ -103,3 +103,19 @@ pub struct UpdateArtifact {
 pub enum UpdateArtifactKind {
     Zone,
 }
+
+/// Sent to a sled agent to request the contents of an Instance's serial console.
+#[derive(Clone, Debug, Deserialize, Serialize, JsonSchema, PartialEq)]
+pub struct InstanceSerialConsoleRequest {
+    /// Character index in the serial buffer (since instance boot) from which to read.
+    pub byte_offset: Option<isize>,
```

Comment:
Is the "since instance boot" here accurate? I can't square that with negative offsets.